kolibri/utils/server.py

```diff
@@ -227,16 +275,15 @@ def configure_http_server(port):
     )
 
     # Mount content files
-    content_files_handler = cherrypy.tools.staticdir.handler(
-        section="/", dir=paths.get_content_dir_path()
-    )
-
-    url_path_prefix = conf.OPTIONS["Deployment"]["URL_PATH_PREFIX"]
-
+    CONTENT_ROOT = "/" + paths.get_content_url(
+        conf.OPTIONS["Deployment"]["URL_PATH_PREFIX"]
+    ).lstrip("/")
+    content_dirs = [paths.get_content_dir_path()] + paths.get_content_fallback_paths()
+    dispatcher = MultiStaticDispatcher(content_dirs)
```

Comment:
In the case where the fallback paths are not used, how much overhead does the multistatic dispatcher add compared to how cherrypy normally handles static files?

One way to be conservative about this would be to only use the MultiStaticDispatcher in the case where there are fallback content paths.

Reply:
It's still using the same cherrypy static handlers. The additional code that's executed per file request is:
https://github.com/learningequality/kolibri/pull/6865/files/78ae3149620cc3e98743ca87968e6557529b95c0#diff-601cde52caf17fa9d77c64ca9e73e927R239-R248

In the case of no fallback content paths, this loop will only be executed once, but it does do an extra `os.path.exists` call in there, which wouldn't be needed in the "single handler" case. Actually, the simplest way to "handle" that might be to just short-circuit the `find_handler` method if there's only one candidate handler anyway.